<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_opex1t">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEX 1T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_capex">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>CAPEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_personal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Personal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_otroopex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Otro OPEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_regresar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregados">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver agregados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregados_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Ultimo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>C1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C16</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Componente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Moneda COP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
